app/Http/Controllers/Service/RegistrationController.php

```diff
@@ -151,38 +152,104 @@ public function edit($id)
      * @param integer $id
      * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
      */
-    public function print($id)
+    public function printOneIndividualFamilyForm($id)
     {
+        // Get User
         $user = Auth::user();
 
+        // Find the Registration and subdata
         $registration = Registration::with([
             'family' => function ($q) {
                 $q->with('children', 'carers');
             }
         ])->findOrFail($id);
 
-        if (!$registration) {
-            // Todo better error handling - but for now, just return is better than 500.
-            // We don't expect this to happen- ever... but
-            return;
-        }
-
-        $carers = $registration->family->carers->all();
+        // Make a filename
+        $filename = 'Registration' . Carbon::now()->format('YmdHis') .'.pdf';
 
-        $params = [
+        // Setup common data
+        $data = [
             'user_name' => $user->name,
             'centre_name' => ($user->centre) ? $user->centre->name : null,
-            'centre' => $registration->centre,
             'sheet_title' => 'Printable Family Sheet',
             'sheet_header' => 'Family Collection Sheet',
+        ];
+
+        $carers = $registration->family->carers->all();
+        $data['regs'][] = [
+            'centre' => $registration->centre,
             'family' => $registration->family,
             'pri_carer' => array_shift($carers),
             // Remove the primary carer from collection
             'sec_carers' => $carers,
             'children' => $registration->family->children,
         ];
 
-        return view('service.printables.family', $params);
+        // throw at a PDF
+        $pdf = PDF::loadView('service.printables.family', $data);
+        $pdf->setPaper('A4', 'landscape');
+        return @$pdf->download($filename);
+
```

Comment:
Extra line. Maybe better before the return instead?

Reply:
agreed